app/views/layouts/application.html.erb

```diff
@@ -7,6 +7,8 @@
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'jquery.js' %>
```

Comment:
Related to the other comment, if you decide to include `application.js` it here to make use of the asset pipeline, you won't need to include `jquery.js` and `jquery_ujs.js` here, since these will be required in the application.js manifest